docs/examples/SPA.md

```diff
@@ -0,0 +1,66 @@
+## Using offline-plugin in a Single Page Application (SPA)
+
+In a SPA, only the inital HTML is served from the server. Any routing beyond that takes place on the client. So how should we configure offline-plugin to handle this?
```

Comment:
This is not really correct. Yes, that's only `index.html` but server serves that for everyone route / nav request / 404 request. Then client loads and decides what to do.